sql/core/src/main/scala/org/apache/spark/sql/internal/CatalogImpl.scala

```diff
@@ -338,7 +371,8 @@ class CatalogImpl(sparkSession: SparkSession) extends Catalog {
       tableType = tableType,
       storage = storage,
       schema = schema,
-      provider = Some(source)
+      provider = Some(source),
+      comment = Some(description)
```

Comment:
@nchammas, BTW why did you choose the name `description` over `comment`? Seems `comment` is more consistent.

Reply:
Hm.. okay, should be fine. At least it's consistent with one side.

Reply:
Just to be clear, are you saying to leave the name as-is in this PR, or to change it to `comment`?